main/src/cgeo/geocaching/activity/Progress.java

```diff
@@ -108,4 +108,11 @@ public synchronized int getProgress() {
     public synchronized void setProgressDivider(final int progressDivider) {
         this.progressDivider = progressDivider;
     }
+
+    public void incrementProgressBy(final int increment) {
+        if (isShowing()) {
+            dialog.incrementProgressBy(increment);
```

Comment:
Why not call `dialog.setProgress(progress)` after having incremented `progress`? This would ensure that `progress` and `Dialog.progress` stay in sync.